src/include/ZividPython/Traits.h

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include <type_traits>
+
+#if __has_include(<experimental/type_traits>)
+#    include <experimental/type_traits>
+#endif
```

Comment:
```suggestion
#else
```

Hide the fallback when not needed.

Reply:
Ah you mean pack the entire implementation (Detail namespace) in the else block?

Reply:
Done